src/request.ts

```diff
@@ -1155,6 +1158,7 @@ export interface SharedQueryOptions {
   };
 }
 export interface RequestOptions extends SharedQueryOptions {
+  databaseId?: string;
```

Comment:
why is this here and not along side wherever `reqOpts.projectId` lives? (looking at `request.ts`, line 1030)

Reply:
`databaseId` is in `RequestOptions` because `reqOpts` is of type `RequestOptions` and `reqOpts` is what gets sent [here](https://github.com/googleapis/nodejs-datastore/blob/4caf162910b2aa2a8f59c9ae26f05d69bab8b7b2/src/request.ts#L1033) to the backend.

`projectId` lives in `sharedQueryOptions` so therefore also lives in `RequestOptions` by inheritance which is why we can attach `projectId` on line 1030. Now the reason `databaseId` should not live in `sharedQueryOptions` is because then they would be included in `RunAggregationQueryRequest`, but they do not need to be. I am not sure why `projectId` lives in `sharedQueryOptions` and perhaps that is something that needs to be investigated further.

Reply:
discussed offline, will move `databaseId` to shared query options to match proto: example https://github.com/googleapis/googleapis/blob/master/google/datastore/v1/datastore.proto#L208-L234